release/changes.md

```diff
@@ -1 +1 @@
-
+* When using the ```project-dir``` argument to specify the build invocation directory, script execution was failing due to wrong ```pushd``` ordering in ```invoke_gradle```
```

Comment:
This is better, but it still leaks implementation details and doesn't really tell the user what was broken. Maybe something like this?

```suggestion
* FIX: Gradle experiment scripts fail when using the ```--project-dir``` argument to specify the build invocation directory.
```